### PR TITLE
Suggestion of improvement for the Allow SProp error message.

### PR DESCRIPTION
**Pull request by Zimmi48**
**Kind:** documentation / user message.

- [ ] Added / updated test-suite
- [x] Corresponding documentation was added / updated (including any warning and error messages added / removed / modified).

### CODE REVIEW

File: doc/sphinx/addendum/sprop.rst [hunk 1]
```diff
@@ -13,21 +13,18 @@ known as strict propositions) as described in
 :cite:`Gilbert:POPL2019`.
 
 Using :math:`\SProp` may be prevented by passing ``-disallow-sprop``
```
Comment (jfehrle) [inline doc/sphinx/addendum/sprop.rst:15]:
```suggestion
Use of :math:`\SProp` may be disabled by passing ``-disallow-sprop``
```

Not incorrect, but IMHO it reads better with "Use of" though I can't cite any grammatical rule for that.

File: doc/sphinx/addendum/sprop.rst [hunk 2]
```diff
@@ -13,21 +13,18 @@ known as strict propositions) as described in
...
 
 .. flag:: Allow StrictProp
    :name: Allow StrictProp
 
-   Allows using :math:`\SProp` when set and forbids it when unset. The
-   initial value depends on whether you used the command line
-   ``-disallow-sprop`` and ``-allow-sprop``.
+   Using :math:`\SProp` is allowed only when this flag is on (this is
+   the default).  It may also be globally turned off with the
+   command-line flag ``-disallow-sprop``.
```
Comment (jfehrle) [inline doc/sphinx/addendum/sprop.rst:24]:
```suggestion
   Enables or disables the use of :math:`\SProp`.  It is enabled by default.
   The command-line flag ``-disallow-sprop`` disables :math:`\SProp` at startup.
```

I assumed you can override -disallow-sprop by setting the flag.

Should that be :math: or just ``` ` ... ` ```?  I guess the whole chapter is that way.

**Review by jfehrle — APPROVED** [review]
Looks good, some suggestions for you to consider.

**Comment by Zimmi48** [comment]
>Should that be :math: or just ` ... `? I guess the whole chapter is that way.

Indeed, in this PR I'm avoiding any change that would be inconsistent with the rest of the chapter.

**Comment by SkySkimmer** [comment]
Should be `|SProp|` I guess

**Comment by Zimmi48** [comment]
I'm pushing an additional commit containing some further improvement suggestions to the SProp documentation.

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
Only the first commit can be backported since Cumulative SProp isn't in 8.11.

**Pull request merged**